index.bs

```diff
@@ -2328,10 +2327,11 @@ An extension can also be an <dfn>authenticator extension</dfn>, meaning that the
 For [=authenticator extensions=], as part of the [=client extension processing=],
 the client also creates the [=authenticator extension input=] value for each extension (often based on the corresponding [=client extension input=] value),
 and passes them to the authenticator in the {{CredentialsContainer/create()}} call (for [=registration extensions=]) or
-the {{CredentialsContainer/get()}} call (for [=authentication extensions=]). These [=authenticator extension input=] values are passed as name-value
+the {{CredentialsContainer/get()}} call (for [=authentication extensions=]). These [=authenticator extension input=] values are represented in [=CBOR=] and passed as name-value
 pairs, with the [=extension identifier=] as the name, and the corresponding [=authenticator extension input=] as the value. The authenticator,
 in turn, performs additional processing for the extensions that it supports, and returns the [=authenticator extension output=] for each as
 specified by the extension.
+The [=authenticator extension output=] is represented in [=CBOR=].
```

Comment:
I think I'd prefer to specify the type of data represented by a term by that term's `<dfn>` so that anyone curious about it just follows the link to get to the type, rather than having to read through the whole rest of the spec.